src/routes/Board/Board.container.tsx

```diff
@@ -251,17 +251,6 @@ export function mergeProps(
 
     const promises: Promise<any>[] = [];
 
-    if (!boardConfig || !boardConfig.creatorUid) {
```

Comment:
I guess this part is still required. It ensures that the board owner is set correctly. This is not covered by any other code changes in this PR.